### DISCUSSION TITLE
Understanding Difference of Simpa and Other Software

### DISCUSSION THREAD

**Discussion by ingmasis** [Ifsttar/I-Simpa#298] [General]
On the Simpa webstie it says "I-Simpa alone is not a calculation software, but is equivalent to a pre and post-processor for acoustic codes. One or more numerical codes must be added in order to obtain a fully functional system."

I am just trying to get my head around what that means, isn't the end result the same as other acoutic software, understanding how much absorbtion, reverberation times, ray tracing with the final model?

**Comment by Picaut** [comment]
It means that I-Simpa was developed in order to be used with any 'external' numerical code. I you have developed your own acoustic code, you could connect it with I-Simpa and use I-Simpa to create sources, receveirs..; run your code and display the results within I-Simpa.

But, I-Simpa is already released with a numerical code, called SPPS, a particle ray-tracing code, similar to classical ray-tracing codes. So you can use I-Simpa as a 'classical' room acoustics software. Most of all physical phenomena are included in SPPS (absorption, diffusion, reflection), excepted diffraction.

Take a look of the documentation:
https://i-simpa-wiki.readthedocs.io/fr/latest/
